jablib/build.gradle.kts

```diff
@@ -238,7 +238,7 @@ dependencies {
     testImplementation("org.wiremock:wiremock-standalone:3.12.1")
 
     // Required for LocalizationConsistencyTest
-    testImplementation("org.testfx:testfx-core:4.0.16-alpha")
+    testImplementation("org.testfx:testfx-core:4.0.18")
     testImplementation("org.testfx:testfx-junit5:4.0.16-alpha")
```

Comment:
this needs to be adpated as well

Reply:
dependabot not working nicely with kotlin? 🙈